upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.13.yaml.template

```diff
@@ -193,6 +193,9 @@ data:
 
   # Name of the cluster. Only relevant when building a mesh of clusters.
   cluster-name: "{{ .ClusterName }}"
+  # ID of the cluster. Only relevant when building a mesh of clusters.
+  # Must be a number between 1 and 255.
+  cluster-id: "{{ .ClusterID }}"
```

Comment:
```suggestion
  {{ if . ClusterID }}
  cluster-id: "{{ .ClusterID }}"
  {{ end }}
```

Reply:
Shouldn't we handle `ClusterID` like `ClusterName` in the Cilium spec?

Reply:
Why set it if there's no mesh involved?

Reply:
True, would it then not be better to only set cluster-name if ClusterName is set // why do we set a default on ClusterName?

They seem to be handled [equally with Cilium](https://docs.cilium.io/en/stable/network/clustermesh/clustermesh/#specify-the-cluster-name-and-id)

Reply:
That ship already sailed. For backwards compatibility it has to stay.

Reply:
Ah okay :+1: